protocols/everclear.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "Everclear",
+    "socials": [
+        {
+            "label": "App",
+            "link": "https://bridge.connext.network/"
+        },
+        {
+            "label": "Website",
+            "link": "https://everclear.org/"
+        },
+        {
+            "label": "Docs",
+            "link": "https://docs.everclear.org/"
+        },
+        {
+            "label": "Twitter",
+            "link": "https://twitter.com/EverclearOrg/"
+        },
+        {
+            "label": "Discord",
+            "link": "https://discord.gg/everclear"
+        },
+        {
+            "label": "Explorer",
+            "link": "https://connextscan.io/"
+        },
+        {
+            "label": "GitHub",
+            "link": "https://github.com/connext"
+        }
+    ],
+    "logo_dark": "/protocols/everclear-dark.png",
+    "logo_light": "/protocols/everclear-light.png",
+    "icon_url": "/protocols/everclear-dark.png",
+    "icon_dark": "/protocols/everclear-dark.png",
+    "icon_light": "/protocols/everclear-light.png",
+    "description": "Everclear is a novel public good mechanism for settling intents across chains.",
+    "mainnet": "yes",
+    "type": "bridge",
+    "chains_supported": "Enables permissionless expansion to new chains and rollups, making it possible for any application on any chain to onboard users from any other chain.",
+    "xchain_sync": "Canonical brdiges for message passing",
+    "auction_design": "RTB",
+    "auction_design_ranking": "Good",
+    "auction_design_details": "Race to the Bottom. Auctioneer smart contract governs filler selection (by honoring the bid with the lowest fees).",
+    "auction_openness": "Good",
```

Comment:
What would we need for this to be excellent? If everyone can see everything always then I think this doesn't get any better?

Reply:
fixed